libcontainer/configs/config.go

```diff
@@ -127,6 +127,12 @@ type Config struct {
 	// If Rlimits are not set, the container will inherit rlimits from the parent process
 	Rlimits []Rlimit `json:"rlimits"`
 
+	// OomScoreAdj specifies the adjustment to be made by the kernel when calculating oom scores
+	// for a process. Valid values are between the range [-1000, '1000'], where processes with
+	// higher scores are preferred for being killed.
+	// More information about kernel oom score calculation here: https://lwn.net/Articles/317814/
```

Comment:
I think could point to https://www.kernel.org/doc/Documentation/filesystems/proc.txt instead as oom_score_adj can't be found on the lwn page.